frontend/src/modules/vehicles/EditVehicleProperties.vue

```diff
@@ -0,0 +1,229 @@
+        </CRow>
+        <hr />
+        <CRow v-if="!!editedVehicle">
+          <CCol>
+            <div
+              v-for="property in leftProperties"
+              :key="property._id"
+              class="mb-2"
+            >
+              <CInput 
+                v-if="property.inputType == 'Text'"
+                :name="property.key"
+                :label="property.label"
+                v-model="editedVehicle[property.key]"
+                horizontal
+                class="mb-0"
+              />
+              <CInput 
+                v-if="property.inputType == 'List'"
+                :name="property.key"
+                :label="property.label"
+                description="Separate list item by semi-colons ';'"
+                v-model="editedVehicle[property.key]"
+                horizontal
+                class="mb-0"
+              />
+              <CSelect
+                v-if="property.inputType == 'Options'"
+                :options="property.options"
+                :name="property.key"
+                :label="property.label"
+                :value.sync="editedVehicle[property.key]"
+                horizontal
+                class="mb-0"
+              />
+              <CInput
+                v-if="property.inputType == 'Date'"
+                :label="property.label"
+                v-model="editedVehicle[property.key]"
+                type="date"
+                horizontal
+              />
+            </div>
+          </CCol>
+          <CCol>
+            <div
+              v-for="property in rightProperties"
+              :key="property._id"
+              class="mb-2"
+            >
+              <CInput 
+                v-if="property.inputType == 'Text'"
+                :name="property.key"
+                :label="property.label"
+                v-model="editedVehicle[property.key]"
+                horizontal
+                class="mb-0"
+              />
+              <CInput 
+                v-if="property.inputType == 'List'"
+                :name="property.key"
+                :label="property.label"
+                description="Separate list item by semi-colons ';'"
+                v-model="editedVehicle[property.key]"
+                horizontal
+                class="mb-0"
+              />
+              <CSelect
+                v-if="property.inputType == 'Options'"
+                :options="property.options"
+                :name="property.key"
+                :label="property.label"
+                :value.sync="editedVehicle[property.key]"
+                horizontal
+                class="mb-0"
+              />
+              <CInput
+                v-if="property.inputType == 'Date'"
+                :label="property.label"
+                v-model="editedVehicle[property.key]"
+                type="date"
+                horizontal
+              />
+            </div>
```

Comment:
This could be refactored into its separate component, however it would become messy since the value needs to be binded in the parent class. We can look for a better solution in sprint 6

Reply:
Okay, I will create an issue for it for the next sprint.